2fa/v1/openapi.yaml

```diff
@@ -19,10 +23,17 @@ security:
   - APIKeyHeader: []
 basePath: /
 tags:
-  - name: 2fa-application-service
-    description: 2FA Application Service
-  - name: 2fa-otp-service
-    description: 2fa Otp Service
+  - name: "Application Service"
+    description: |
+      The 2FA API has a part where the user can create different 2FA applications and separate his configuration.
+      tyntec stores these configurations for the user and the user can refer to the applicationId that points to 
+      the desired configuration that will be applied for the specific OTP delivery.
```

Comment:
What is an OTP? Might makes sense to write it On Time Password (OTP)

Reply:
OK, i will add this.